examples-gallery/two-pendulum-cart-variable-gallery.py

```diff
@@ -0,0 +1,311 @@
+"""
+Upright a double pendulum.
+==========================
+
+A double pendulum is rotably attached to a cart which can move
+along the horizontal X axis. The goal is to get the double
+pendulum to an upright position in the shortest time possible,
+given an upper limit on the absolute value of the force that
+can be applied to the cart.
+Gravity points in the negative Y direction.
+
+**Constants**
+
+m1:     mass of the cart [kg]
+m2:     mass of the first pendulum [kg]
+m3:     mass of the second pendulum [kg]
+lx:     length of the first pendulum rods [m]
+iZZ1:   moment of inertia of the first pendulum about its end point [kg*m^2]
+iZZ2:   moment of inertia of the second pendulum about its end point [kg*m^2]
+g:      acceleration due to gravity [m/s^2]
+
+**States**
+
+q1:     position of the cart along the X axis [m]
+q2:     angle of the first pendulum with respect to the vertical [rad]
+q3:     angle of the second pendulum with respect to the first [rad]
+u1:     speed of the cart along the X axis [m/s]
+u2:     angular speed of the first pendulum [rad/s]
+u3:     angular speed of the second pendulum [rad/s]
+
+**Specifieds**
+
+f:      force applied to the cart [N]
+
+"""
+import sympy.physics.mechanics as me
+from collections import OrderedDict
+import numpy as np
+import sympy as sm
+from opty.direct_collocation import Problem
+
+import matplotlib.pyplot as plt
+import matplotlib
+import matplotlib.animation as animation
+matplotlib.rcParams['animation.embed_limit'] = 2**128
+from matplotlib import patches
+
+# %%
+# Generate the nonholonomic equations of motion of the system.
+N, A1, A2      = sm.symbols('N A1, A2', cls = me.ReferenceFrame)
```

Comment:
```suggestion
N, A1, A2 = sm.symbols('N A1, A2', cls=me.ReferenceFrame)
```

Reply:
See the python style guide for things like this: https://peps.python.org/pep-0008/

The only style we don't follow is the capital variable names for frames and points, those pep 8 is relaxed for the sake of fitting with common math notation.